docs/documentation/form/file.html

```diff
@@ -527,6 +527,16 @@
 </div>
 {% endcapture %}
 
+{% capture required_input_file %}
+<form>
+  <input
+    type="file"
```

Comment:
You need the class as well I assume.

Reply:
Well, actually I wanted to just present the code with `required` over the `input type=file`. Didn't mean to use bulma specific code in here. It is available above anyway, so just wanted to back up the paragraph above with very short code snippet.